index.html

```diff
@@ -3640,11 +3641,11 @@ <h5>JWT Decoding</h5>
 Create a JSON object.
               </li>
               <li>
-Add the content from the <code>vc</code> <a>claim</a> to the new JSON object.
+Add the content from the <code>vc</code> or <code>vp</code> <a>claim</a> to the new JSON object.
               </li>
               <li>
 Transform the remaining JWT specific headers and <a>claims</a>, and add the
-results to the new JSON object.
+results to the new <a>credential</a> or <a>presentation</a> JSON object.
```

Comment:
this is so great to see! I wonder if we might fully define these transformation functions, I would like to see them made explicit.